cmd/export.go

```diff
@@ -33,12 +38,25 @@ func setupExportCommand() *cobraext.Command {
 	exportDashboardCmd.Flags().StringSliceP(cobraext.DashboardIDsFlagName, "d", nil, cobraext.DashboardIDsFlagDescription)
 	exportDashboardCmd.Flags().Bool(cobraext.TLSSkipVerifyFlagName, false, cobraext.TLSSkipVerifyFlagDescription)
 
+	exportInstalledObjectsCmd := &cobra.Command{
+		Use:   "installed-objects",
+		Short: "Export installed Elasticsearch objects",
+		Long:  exportInstalledObjectsLongDescription,
+		RunE:  exportInstalledObjectsCmd,
+	}
+	exportInstalledObjectsCmd.Flags().StringP(cobraext.ExportPackageFlagName, "p", "", cobraext.ExportPackageFlagDescription)
+	exportInstalledObjectsCmd.MarkFlagRequired(cobraext.ExportPackageFlagName)
```

Comment:
With the approach here you can export the objects without needing the source of the package, I think this will be useful for the current use case, and it actually doesn't need anything about the current package context.

If we want to make all `export` subcommands to be consistent on this I think I would prefer to add both options to both subcommands (i.e. both work with the package context and with the package flag) :)

Reply:
> If we want to make all export subcommands to be consistent on this I think I would prefer to add both options to both subcommands (i.e. both work with the package context and with the package flag) :)

Yes, I was looking at low hanging fruits :) I will leave you to make the decision.

Reply:
I have added a test so this continues being a low hanging fruit in the future, but I am not taking it by now :)